README.md

```diff
@@ -0,0 +1,71 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Space Travelers' Hub
+
+> Space Travelers' Hub is a website for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+## Tools & Technologies
+
+- React.
+- Redux.
+- Redux Toolkit.
+- React Router.
+- React Hooks.
+- React Testing Library.
+- Jest.
+- Eslint.
+- Stylelint.
+
+## Live Demo 
+
+[Click Me!](https://spacexova.netlify.app)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+```
+git clone git@github.com:OmarMWarraich/Space-Travelers-Hub.git
+cd Space-Travelers-Hub
+npm install
+npm start
+```
```

Comment:
- [Optional] Your README file is professional 💯 However, I suggest adding instruction for testing "npm run test" or "npm test" 👍